services/src/main/java/com/google/daq/mqtt/mapping/MappingService.java

```diff
@@ -1,102 +1,126 @@
 package com.google.daq.mqtt.mapping;
 
+import static com.google.bos.iot.core.reconcile.SourceRepoMessageUtils.parseSourceRepoMessageData;
+import static com.google.udmi.util.GeneralUtils.isNotEmpty;
 
 import com.google.daq.mqtt.registrar.Registrar;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.udmi.util.AbstractPollingService;
 import com.google.udmi.util.SourceRepository;
-import java.nio.file.Paths;
+import java.io.IOException;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * Mapping Service: which currently runs from command line
+ * Mapping Service basically
```

Comment:
replace `basically` with `that`, I think -- otherwise it reads oddly informal...